tests-ng/test_basics.py

```diff
@@ -57,3 +57,15 @@ def test_startup_time(systemd: Systemd):
         f"Userspace startup too slow: {userspace:.1f}s "
         f"(tolerated {tolerated_userspace}s)"
     )
+
+@pytest.mark.feature("not gcp and not metal and not openstack", reason='Not working on various features, for example because the vm cannot communicate with external backends.')
+@pytest.mark.root(reason="Needed for journalctl which is only needed when the test fails, but still very useful for understanding test failures")
```

Comment:
While I like the general approach of reasons I'm not sure if we add too much information / text here. Could we sum these texts up to be as short as possible? E.g.: `Not compatible, usually because of missing external backend communication channel` or `Required for journalctl in case if errors`